nextflow.config

```diff
@@ -103,7 +108,7 @@ profiles {
         shifter.enabled         = false
         charliecloud.enabled    = false
         apptainer.enabled       = false
-        docker.runOptions       = '-u $(id -u):$(id -g)'
+        // docker.runOptions       = '-u $(id -u):$(id -g)'
```

Comment:
Had to comment this out to make it work, due to permissions within the benchmark docker image. If this config is still required for some reason, I can go and add a user to modify the docker image, otherwise can we maybe remove this?

Reply:
I think https://github.com/nextflow-io/nextflow/issues/1143. We could just set this in that process.